models/organization/src/team/generator.ts

```diff
@@ -1,14 +1,12 @@
 import { Reference } from '@commercetools-test-data/commons';
 import { fake, Generator } from '@commercetools-test-data/core';
-import { User } from '@commercetools-test-data/user';
-import type { TTeam } from './types';
+import type { TTeam, TTeamRest } from './types';
 
-const generator = Generator<TTeam>({
+const generator = Generator<TTeam | TTeamRest>({
```

Comment:
We should use only the default type 

```suggestion
const generator = Generator<TTeam>({
```